setup.cfg

```diff
@@ -10,7 +10,6 @@ project_urls =
 author = Holger Krekel, Bruno Oliveira, Ronny Pfannschmidt, Floris Bruynooghe, Brianna Laugher, Florian Bruhin and others
 
 license = MIT license
-license_file = LICENSE
```

Comment:
IIRC `LICENSE` is added automatically when found in newer setuptools, correct?

Reply:
It'll be automatically included in sdists by setuptools-scm (https://github.com/pytest-dev/pytest/pull/6348#issuecomment-567836331), and by wheel in wheels (https://wheel.readthedocs.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file).

Reply:
Thanks for the confirmation! 👍